bundling.cmake

```diff
@@ -6,7 +6,7 @@ endif()
 
 # FIXME (aw): our packaging does not play well with lib64 and these kind of
 #             variations.  Either we need to change our directory layout or ...?
-set(CMAKE_INSTALL_LIBDIR lib)
+# set(CMAKE_INSTALL_LIBDIR lib)
```

Comment:
Can't this whole section be removed entirely then?

Reply:
Changed and removed the section